org.gnome.Epiphany.json

```diff
@@ -1,19 +1,19 @@
 {
     "app-id": "org.gnome.Epiphany",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "42",
+    "runtime-version": "43",
     "sdk": "org.gnome.Sdk",
     "command": "epiphany",
     "finish-args": [
         "--device=dri",
-        "--filesystem=/run/.heim_org.h5l.kcm-socket",
         "--filesystem=xdg-download",
         "--share=ipc",
         "--share=network",
         "--socket=fallback-x11",
         "--socket=pulseaudio",
         "--socket=wayland",
-        "--system-talk-name=org.freedesktop.GeoClue2"
+        "--system-talk-name=org.freedesktop.GeoClue2",
+        "--own-name=org.gnome.Epiphany.WebAppProvider"
```

Comment:
You are given permission to `org.gnome.Epiphany.*` by default.

Reply:
Hmm, maybe that was needed for Tech Preview because there the app ID is `org.gnome.Epiphany.Devel`....

Reply:
Indeed.